naming/src/main/java/com/alibaba/nacos/naming/push/v2/executor/PushExecutor.java

```diff
@@ -30,19 +31,21 @@
     /**
      * Do push.
      *
+     * @param service    the origin service
      * @param clientId   client id
      * @param subscriber subscriber
      * @param data       push data
      */
-    void doPush(String clientId, Subscriber subscriber, PushDataWrapper data);
+    void doPush(Service service, String clientId, Subscriber subscriber, PushDataWrapper data);
```

Comment:
Add Service into PushDataWrapper will be better.

Reply:
OK, I will move it later.